src/main/java/com/redhat/devtools/intellij/commonuitest/fixtures/mainidewindow/idestatusbar/IdeStatusBar.java

```diff
@@ -65,7 +65,14 @@ public void waitUntilProjectImportIsComplete() {
      * Wait until all the background tasks finish
      */
     public void waitUntilAllBgTasksFinish() {
-        waitFor(Duration.ofSeconds(600), Duration.ofSeconds(10), "The background tasks did not finish in 10 minutes.", this::didAllBgTasksFinish);
+        waitUntilAllBgTasksFinish(300);
+    }
+
+    /**
+     * Wait until all the background tasks finish
+     */
+    public void waitUntilAllBgTasksFinish(int timeout) {
+        waitFor(Duration.ofSeconds(timeout), Duration.ofSeconds(10), "The background tasks did not finish in 10 minutes.", this::didAllBgTasksFinish);
```

Comment:
The error message should change to `timeout` seconds, right? It is not 10 minutes anymore.

Reply:
You are right, I have overlooked that. But anyway - I have decided to delete the entire method after all, the test failing due to the timeout is back green. There must be some minor updates from JetBrains to the version 2022.1 that made the test failing and also made it green again.

Reply:
Update: it looks like the issue with timeout is back again. I tried to increase timeout to 400 (only in the one place, so the parametrized method will be handy - with fixed error message)